astroquery/mast/cutouts.py

```diff
@@ -204,8 +223,8 @@ def get_sectors(self, *, coordinates=None, radius=0*u.deg, objectname=None, movi
             warnings.warn("Coordinates are not in any TESS sector.", NoResultsWarning)
         return Table(sector_dict)
 
-    def download_cutouts(self, *, coordinates=None, size=5, sector=None, path=".", inflate=True,
-                         objectname=None, moving_target=False, mt_type=None, verbose=False):
+    def download_cutouts(self, *, coordinates=None, size=5, sector=None, product='SPOC', path=".",
+                         inflate=True, objectname=None, moving_target=False, mt_type=None, verbose=False):
```

Comment:
`inflate` is not described in the docstring

Reply:
Inflate is in the docstring, starting at L235

Reply:
Ahhh, I see, it was _already_ in the docstring but wasn't in the parameters.  Got it!